packages/help-center/src/components/help-center-article.tsx

```diff
@@ -20,18 +20,23 @@ export const HelpCenterArticle = () => {
 	const { data: post, isLoading, error } = usePostByUrl( postUrl );
 
 	useEffect( () => {
-		//If a url includes an anchor, let's scroll this into view!
+		let timeoutId: number;
 		if ( postUrl?.includes( '#' ) && post?.content ) {
-			setTimeout( () => {
-				const anchorId = postUrl.split( '#' ).pop();
-				if ( anchorId ) {
+			const anchorId = postUrl.split( '#' ).pop();
+			if ( anchorId ) {
+				timeoutId = setTimeout( () => {
```

Comment:
The type checking fails here: ` packages/help-center/src/components/help-center-article.tsx(27,5): error TS2322: Type 'Timeout' is not assignable to type 'number'.`

https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/13156866?buildTab=log&linesState=529&logView=flowAware&expandAll=true&focusLine=567